### PR TITLE
Migrate to 2021 edition

### PR DESCRIPTION
**Pull request by kchibisov**

### CODE REVIEW

File: src/github.rs [hunk 1]
```diff
@@ -60,7 +60,7 @@ impl Notification {
     /// This will return an error if the subscription's subject is not a pull request.
     pub fn pull_request(&self) -> Result<PullRequest, Box<dyn Error>> {
         if self.subject.r#type != SubjectType::PullRequest {
-            return Err("subscription is not a pull request")?;
+            return Err("subscription is not a pull request".into());
```
Comment (chrisduerr) [inline src/github.rs:63]: Is that an edition idiom warning? Seems surprising to me.

File: src/github.rs [hunk 2]
```diff
@@ -167,6 +167,7 @@ struct CommentRequest {
 }
 
 /// User's association with a repository.
+#[allow(clippy::upper_case_acronyms)]
```
Comment (chrisduerr) [inline src/github.rs]: Should probably use serde's rename options?

File: src/model.rs [hunk 3]
```diff
@@ -84,5 +83,5 @@ impl NewJob {
 pub fn db_connection() -> SqliteConnection {
     let database_url = env::var("DATABASE_URL").expect("DATABASE_URL environment variable missing");
     SqliteConnection::establish(&database_url)
-        .expect(&format!("Unable to find DB: {}", database_url))
+        .unwrap_or_else(|_| panic!("Unable to find DB: {}", database_url))
```
Comment (chrisduerr) [inline src/model.rs:86]: The hell is this?

**Review by chrisduerr — CHANGES_REQUESTED** [review]

File: src/model.rs (hunk 3 above)
Comment (kchibisov) [reply src/model.rs:86]: Well, ask clippy.

File: src/github.rs (hunk 1 above)
Comment (kchibisov) [reply src/github.rs:63]: I'm not sure, just clippy warned about this.

File: src/model.rs (hunk 3 above)
Comment (kchibisov) [reply src/model.rs:86]: But from what I can see it's to avoid evaluation of `format!`.

**Pull request closed without merging**